app/views/medicaid/contact_current_address/edit.html.erb

```diff
@@ -0,0 +1,27 @@
+<% content_for :header_title, "Contact Information & Followup" %>
```

Comment:
I renamed this page, so this can be removed now :)

Reply:
@jessieay do you mean this page should be deleted?

Reply:
yep, I removed in previously and you re-added it here but it is not used